nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/ui/reporting-tasks/edit-reporting-task/edit-reporting-task.component.html

```diff
@@ -15,7 +15,7 @@
   ~ limitations under the License.
   -->
 
-<h2 mat-dialog-title>Edit Reporting Task</h2>
+<h2 mat-dialog-title>Configure Reporting Task</h2>
```

Comment:
I think most of the edit dialogs are called `EditXyz`, the titles are `Edit Xyz`, the form is named `editXyz`, etc. Also, the routes use `edit` and the action names include `edit`. It's possible that there are some instances of `Configure` that are present (like in the Context Menu)  but if we're looking to make this more consistent there are many places that will need to be updated.

Reply:
Not sure but current NiFi shows 'Configure Reporting Task' so that is what I went with:

![Screenshot 2024-01-11 at 5 05 29 PM](https://github.com/apache/nifi/assets/6797571/a9f51ea6-96c8-4e76-b43c-13b19420176f)

Reply:
Ok we will go with Edit in dialog titles.